lib/idl/autocrat_v0.json

```diff
@@ -8,7 +8,7 @@
         {
           "name": "dao",
           "isMut": true,
-          "isSigner": false
+          "isSigner": true
```

Comment:
You end up with a bunch of IDLs over the lifetime of the we'll have IDLs for v0 v0.1 v0.2 and v0.3. Hence the naming convention (even though the program changes and updates, it's always the most recent deploy in the repo.

Reply:
Okay cool I added v0-v0.3 so now we should have things lined up. I anchor built `deploy` for v0.3 and anchor build `production` for v0.2 and the program IDs lined up in that regard.